draft-ietf-rtcweb-jsep.xml

```diff
@@ -2779,6 +2773,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             NOT contain any ICE credentials.</t>
 
             <t>If the m= section is not bundled into another m=
+            section and RTCP multiplexing is not active, an "a=rtcp" attribute
```

Comment:
'depending on whether' clause is unnecessary here. 

Also, because you already discussed adding the a=rtcp line above, you can just say 'the "a=rtcp" attribute MUST be filled in [...]', rather than 'MUST be added'.

Lastly, you probably want to say "If no **RTCP** candidates have yet been gathered [...]"